priv/cabbage/apps/itest/test/features/invalid_standard_exits.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Invalid Standard Exits
+  Scenario: Alice starts an invalid Standard Exit
+    Given Alice deposited "12" ETH on the child chain
+    And The child chain is secure
+    When Alice sends Bob "11" ETH on the child chain
+    And Alice starts a standard exit on the child chain from her recently spent input
+    But Bob detects a new "invalid_exit"
+    And Bob challenges an invalid exit
+    And Alice tries to process exits
+    Then The child chain is secure
+    And Alice should have "12" ETH less on the blockchain
+
+  Scenario: Alice starts an invalid Standard Exit from a non-deposit
+    Given Alice received "12" ETH on the child chain
+    And The child chain is secure
+    When Alice sends Bob "11" ETH on the child chain
+    And Alice starts a standard exit on the child chain from her recently spent input
+    But Bob detects a new "invalid_exit"
+    And Bob challenges an invalid exit
+    And Alice tries to process exits
+    Then The child chain is secure
+    And Alice should have "0" ETH less on the blockchain
```

Comment:
For the readability we should have a counter `should have "X" ETH more` 
also `r: blockchain/root chain`

Reply:
I see this is slightly weird, but there is also the gas and plasma-tx fee being deducted and considered in this step. So phrasing this optimally I would say is up to a new PR.